toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -317,14 +339,15 @@ func findSPECFiles(specsDir string, packList []string) (specFiles []string, err
 }
 
 // createChroot creates a chroot to pack SRPMs inside of.
-func createChroot(workerTar, buildDir, outDir, specsDir string) (chroot *safechroot.Chroot, newBuildDir, newOutDir, newSpecsDir string, err error) {
+func createChroot(workerTar, buildDir, outDir, specsDir, cacheDir string) (chroot *safechroot.Chroot, newBuildDir, newOutDir, newSpecsDir, newCacheDir string, err error) {
```

Comment:
Similar comment: let's call the new argument `sourcesCacheDir`. Otherwise it's unclear what the cache holds.